### ISSUE TITLE
Registering cell renderers and cell formatters seems to be ignored

### ISSUE THREAD

**Issue by bleurubin** [jameskleeh/groovy-excel-builder#7]
Cell renderers and cell formatters registered as below seem to be ignored:

  ```
Excel.registerCellRenderer(String, 0) { it + 'extra' }
Excel.registerCellFormat(BigDecimal, 0xa)
```

I am only able to format cells using the [format: 0xa] style within a cell:

`cell(new BigDecimal("1.32"), [format: 0xa])`

**Comment by jameskleeh** [comment]
Thanks for reporting. Looks like if you provided any options at all it would work. I'm working on a fix and will release ASAP

**Comment by jameskleeh** [comment]
Fixed and released 0.3.0

**Comment by bleurubin** [comment]
Wow- thanks for such a quick turnaround! The DSL looks great btw.
